### PR TITLE
Feature/linked shared activity enhancements

### PR DESCRIPTION
**Pull request by arvindgh**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [x] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [x] Sanity Tests are successfully executed for Existing Functionality
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments

### CODE REVIEW

File: Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs [hunk 1]
```diff
@@ -101,5 +109,23 @@ private void SelectBtn_Click(object sender, RoutedEventArgs e)
...
+            xNoteLable.Content = "Note- * If we update any instance of this type, all the flows having this instance will be updated.";
+        }
+
+        private void xRegularInstance_Checked(object sender, RoutedEventArgs e)
+        {
+            if (xNoteLable == null)
+            {
+                return;
+            }
+            xNoteLable.Content = "Note- * If we update any instance of this type, we need to manually updated all the flows with this instance.";
```
Comment (Maheshkale447) [inline Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs:128]: change line to below 
Note - Any updation to instance of this type, needs manual updation of all the flows with this instance.

File: Ginger/Ginger/Variables/VariablesPage.xaml [hunk 2]
```diff
@@ -1,20 +0,0 @@
-    <Page
```
Comment (Maheshkale447) [inline Ginger/Ginger/Variables/VariablesPage.xaml:1]: Why this class was deleted?

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/Variables/VariablesPage.xaml (hunk 2 above)
Comment (arvindgh) [reply Ginger/Ginger/Variables/VariablesPage.xaml:1]: Not sure, @Jinendra Can you please confirm?

File: Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs [hunk 3]
```diff
@@ -101,5 +109,23 @@ private void SelectBtn_Click(object sender, RoutedEventArgs e)
...
             mPageGenericWin.Close();
         }
+
+        private void xLinkedInstance_Checked(object sender, RoutedEventArgs e)
+        {
+            if (xNoteLable == null)
+            {
+                return;
+            }
+            xNoteLable.Content = "Note- * If we update any instance of this type, all the flows having this instance will be updated.";
```
[the remark this answers is not in the document]
Comment (arvindgh) [reply Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs]: Done

File: Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs (hunk 1 above)
Comment (arvindgh) [reply Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**